.github/workflows/snyk-pr.yml

```diff
@@ -0,0 +1,31 @@
+name: snyk-pr
```

Comment:
This file sounds not related to this PR.

Reply:
OK